RecoLocalTracker/SiPixelClusterizer/plugins/gpuClustering.h

```diff
@@ -274,7 +274,7 @@ namespace gpuClustering {
       // adjust the cluster id to be a positive value starting from 0
       for (int i = first; i < msize; i += blockDim.x) {
         if (id[i] == invalidModuleId) {  // skip invalid pixels
-          clusterId[i] = -9999;
+          clusterId[i] = -gpuClustering::invalidModuleId;
```

Comment:
there is probably no harm, but the named constant about a module ID likely does not apply to a cluster index.
I've seen this one earlier during the review and considered to propose `SiPixelCluster::invalidClusterId`, but then dropped due to not very obvious application/case.
I suggest to revert back to `-9999` or use `SiPixelCluster::invalidClusterId`

Reply:
Maybe `-SiPixelCluster::invalidClusterId` - I think it has to be negative.

Reply:
Ah, but `SiPixelCluster::invalidClusterId` now is `std::numeric_limits<uint16_t>::max()`, so making it negative doesn't seem a good idea.

Reply:
> Maybe `-SiPixelCluster::invalidClusterId` - I think it has to be negative.

agreed.

clusterId is int32_t; so, the negation of `std::numeric_limits<uint16_t>::max()` in this context is numerically still meaningful.

anyways, it seems semantically safer to stay with just `-9999`